### PR TITLE
Exhaust reports config update

### PR DESCRIPTION
**Pull request by manjudr**

### CODE REVIEW

File: ansible/roles/data-products-deploy/templates/common.conf.j2 [hunk 1]
```diff
@@ -245,3 +245,17 @@ dcedialcode.filename="DCE_dialcode_data.csv"
 etbdialcode.filename="ETB_dialcode_data.csv"
 dcetextbook.filename="DCE_textbook_data.csv"
 etbtextbook.filename="ETB_textbook_data.csv"
+
+## Collection Exhaust Jobs Configuration - Start ##
+
+sunbird.user.keyspace="sunbird"
+sunbird.courses.keyspace="sunbird_courses"
+sunbird.user.keyspace="sunbird"
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/data-products-deploy/templates/common.conf.j2:253]: This is a duplicate entry

File: ansible/roles/data-products-deploy/templates/common.conf.j2 [hunk 2]
```diff
@@ -245,3 +245,17 @@ dcedialcode.filename="DCE_dialcode_data.csv"
 etbdialcode.filename="ETB_dialcode_data.csv"
 dcetextbook.filename="DCE_textbook_data.csv"
 etbtextbook.filename="ETB_textbook_data.csv"
+
+## Collection Exhaust Jobs Configuration - Start ##
+
+sunbird.user.keyspace="sunbird"
+sunbird.courses.keyspace="sunbird_courses"
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/data-products-deploy/templates/common.conf.j2:252]: can you parameterize the values?

**Review by SanthoshVasabhaktula — CHANGES_REQUESTED** [review]

File: ansible/roles/data-products-deploy/templates/model-config.j2 [hunk 3]
```diff
@@ -37,6 +37,15 @@ config() {
...
+		;;
+		"progress-exhaust")
+		echo '{"search":{"type":"none"},"model":"org.sunbird.analytics.exhaust.collection.ProgressExhaustJob", "modelParams":{"store": "azure", "mode":"OnDemand", "batchFilters":["TPD"],"applyPrivacyPolicy":true,"contentFilters":{},"reportPath":"progress-exhust/","fromDate":"$(date --date yesterday '+%Y-%m-%d')","toDate":"$(date --date yesterday '+%Y-%m-%d')","sparkCassandraConnectionHost":"'$sunbirdPlatformCassandraHost'","sparkElasticsearchConnectionHost":"'$sunbirdPlatformElasticsearchHost'","sparkRedisConnectionHost":"'$sparkRedisConnectionHost'","sparkUserDbRedisIndex":"12"},"output":[{"to":"console","params":{"printEvent":false}}],"parallelization":8,"appName":"Progress Exhaust","deviceMapping":false}'
+		;;
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/data-products-deploy/templates/model-config.j2:48]: Fix the config Manju. Cassandra host, batchfilter, output, search, etc are not required

**Review by SanthoshVasabhaktula — CHANGES_REQUESTED** [review]

File: ansible/roles/data-products-deploy/templates/model-config.j2 (hunk 3 above)
Comment (manjudr) [reply ansible/roles/data-products-deploy/templates/model-config.j2]: `batchfilter` is required we are using in the `standalone` method 
`val batchId = modelParams.get("batchId").asInstanceOf[Option[String]];
    val batchFilter = modelParams.get("batchFilter").asInstanceOf[Option[List[String]]];
    val searchFilter = modelParams.get("searchFilter").asInstanceOf[Option[Map[String, AnyRef]]];
   `

**Review by SanthoshVasabhaktula — APPROVED** [review]

**Pull request merged**